lms/models/application_instance.py

```diff
@@ -142,11 +143,11 @@ def update_lms_data(self, params: dict):
         ):
             # If we already have a LMS guid linked to the AI
             # and we found a different one report it to sentry
-            LOG.exception(
-                "Application Instance ID:%s launched in a different LMS install",
-                self.id,
+            raise ReusedToolGuidError(
+                "Application Instance launched in a different LMS install",
```

Comment:
I think I'd prefer this text to be inside the exception itself as it acts as good documentation of what the exception represents. Either that or add a docstring?

If there's only one possible value for this it's kind of nice to tuck the string generation into the exception I think.

Reply:
Yup,that message is a bit pointless if it's always the same, It made a little bit of sense when it was a `ValueError`. Updated.